documentation/modules/exploit/linux/http/panos_auth_rce.md

```diff
@@ -0,0 +1,47 @@
+This module exploits an authenticated OS Command Injection vulnerability in PAN-OS versions: <10.0.1, <9.1.4 and <9.0.10
+
+## Vulnerable Application
+
+A Palo Alto Firewall demo can be requested at the following [link](https://www.paloaltonetworks.com/company/request-demo)
+
```

Comment:
Umm isn't this the firewall and not the OS? Might need some more info here to tie this into how you are meant to use this to get access to PAN-OS itself as the tie here is not obvious. I am also pointing this out as I'm not sure if this was left over from another module given the current explanation, but feel free to correct me if I am wrong.

Reply:
I did write this. PAN‑OS is the software that runs all Palo Alto Networks next-generation firewalls. I was able to get a copy of a VM internally. I believe if someone wants to obtain a copy of PAN-OS they have to sign up and request a demo. I'll add a bit more detail here.